src/components/headersEditor/HeadersEditor.tsx

```diff
@@ -0,0 +1,109 @@
+import { CloseSquareFilled, DeleteOutlined, PlusOutlined } from '@ant-design/icons';
+import { AutoComplete, Button, Checkbox, Divider, Form, Space } from 'antd';
+import React, { useState } from 'react';
+
+import { defaultHeaderValues } from '@/constants/defaultHeaderValues';
+import { Header, HeadersEditorProps, HeadersFormValues } from '@/types/headersSectionTypes';
+
+export default function HeadersEditor({ onChange }: Readonly<HeadersEditorProps>) {
+  const [options] = useState<{ value: string }[]>(
+    Object.keys(defaultHeaderValues).map((header) => ({ value: header }))
+  );
```

Comment:
здесь точно нужен useState? setOptions  не определён, мб просто в useMemo определить значение options'ов?

```suggestion
  const options = useMemo(
    () => Object.keys(defaultHeaderValues).map((header) => ({ value: header })),
    []
  );
```

Reply:
исправила)